### PR TITLE
init-site.sh fixes:

### PR DESCRIPTION
**Pull request by avasquez614**
* Fixed several problems with init-site.sh, mainly the problem with just calling it with one param (fix for craftercms/craftercms#1605)

### CODE REVIEW

File: resources/crafter/init-site.sh [hunk 1]
```diff
@@ -5,77 +5,64 @@
...
+if [[ ! "$REPO" =~ ^ssh.* ]] && [ ! -d "$REPO" ]; then
+	echo -e "\033[38;5;196m"
+	echo -e "Repository path $REPO for site \"$SITE\" does not exist or cannot to read"
+	echo -e "\033[0m"
+	exit 4
+fi
 
 echo "Creating Solr Core"
-curl -s -X POST -H "Content-Type: application/json" -d '{"id":"'"$SITE"'"}' "http://localhost:@TOMCAT_HTTP_PORT@/crafter-search/api/2/admin/index/create"
+curl -s -X POST -H "Content-Type: application/json" -d '{"id":"'"$SITE"'"}' "http://localhost:9080/crafter-search/api/2/admin/index/create"
```
Comment (cortiz) [inline resources/crafter/init-site.sh]: Do not hardcode ports use variables (in crafter-setenv.sh or the ones filtered by gradle)

File: resources/crafter/init-site.sh [hunk 2]
```diff
@@ -5,77 +5,64 @@
...
 echo ""
 
 echo "Creating Deployer Target"
-curl -s -X POST -H "Content-Type: application/json" -d '{"env":"default", "site_name":"'"$SITE"'", "template_name":"remote", "repo_url":"'"$REPO"'", "repo_branch":"live", "engine_url":"http://localhost:@TOMCAT_HTTP_PORT@" '$PRIVATE_KEY' }' "http://localhost:@DEPLOYER_PORT@/api/1/target/create"
+curl -s -X POST -H "Content-Type: application/json" -d '{"env":"default", "site_name":"'"$SITE"'", "template_name":"remote", "repo_url":"'"$REPO"'", "repo_branch":"live", "engine_url":"http://localhost:9080" '$PRIVATE_KEY' }' "http://localhost:9192/api/1/target/create"
```
Comment (cortiz) [inline resources/crafter/init-site.sh]: Same as before, no hardcoded ports

File: resources/crafter/init-site.sh (hunk 1 above)
Comment (avasquez614) [reply resources/crafter/init-site.sh]: @cortiz Fixed with the latest commit

File: resources/crafter/init-site.sh (hunk 2 above)
Comment (avasquez614) [reply resources/crafter/init-site.sh]: @cortiz Fixed with the latest commit

**Pull request merged**